radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/substatehash/BerkeleySubstateAccumulatorHashStore.java

```diff
@@ -173,7 +175,8 @@ public void open(DatabaseEnvironment dbEnv) {
         getPreviousSubStateAccumulatorHash().orElse(HashUtils.zero256().asBytes());
     this.lastEpochInDbOpt = getLatestStoredEpoch();
     this.lastStateVersionInDbOpt = getPreviousSubStateAccumulatorStateVersion();
-    try (BufferedReader bufferedReader = openEpochsHashFileAsRead(EPOCH_HASH_FILE_READ_PATH)) {
+    try (BufferedReader bufferedReader =
+        openEpochsHashFileAsRead(File.separator + getEpochHashFilenameForCurrentNetwork())) {
```

Comment:
`openEpochsHashFileAsRead` has a different signature to `openEpochsHashFileAsAppend` - I think I'd prefer the latter signature - ie that `openEpochsHashFileAsRead()` will compute the path to the file inside the method itself, to reduce code duplication

Reply:
Makes sense ;)